src/main/java/org/opentripplanner/middleware/tripmonitor/jobs/CheckMonitoredTrip.java

```diff
@@ -329,6 +319,44 @@ private boolean makeOTPRequestAndUpdateMatchingItineraryInternal() {
         return false;
     }
 
+    /** Default implementation for OtpResponse provider that actually invokes the OTP server. */
+    private OtpResponse getOtpResponse() {
+        OtpDispatcherResponse otpDispatcherResponse;
+        try {
+            // Generate the appropriate OTP query params for the trip for the current check by replacing the date query
+            // parameter with the appropriate date.
+            Map<String, String> params = trip.parseQueryParams();
+            params.put(ItineraryUtils.DATE_PARAM, targetZonedDateTime.format(DateTimeUtils.DEFAULT_DATE_FORMATTER));
+            otpDispatcherResponse = OtpDispatcher.sendOtpPlanRequest(OtpVersion.OTP1, ItineraryUtils.toQueryString(params));
+        } catch (Exception e) {
+            BugsnagReporter.reportErrorToBugsnag(
+                "Encountered an error while making a request ot the OTP server.",
+                e
+            );
+            return null;
+        }
+
+        // FIXME in PR Review: Is the below if statement needed? (SonarLint thinks not.)
+        if (otpDispatcherResponse == null) return null;
```

Comment:
Should we keep this check?

Reply:
Removed in https://github.com/ibi-group/otp-middleware/pull/228/commits/319186984e8d111f4ed8a44b916ae53c9c2b5339